src/app/modules/submission/encrypt-submission/encrypt-submission.controller.ts

```diff
@@ -647,13 +663,209 @@ const submitEncryptModeForm: ControllerHandler<
   )
 }
 
+// TODO (FRM-1232): remove endpoint after encryption boundary is shifted
 export const handleEncryptedSubmission = [
   CaptchaMiddleware.validateCaptchaParams,
   TurnstileMiddleware.validateTurnstileParams,
   EncryptSubmissionMiddleware.validateEncryptSubmissionParams,
   submitEncryptModeForm,
 ] as ControllerHandler[]
 
+/**
+ * Reference implementation taken from frontend/src/features/public-form/utils/createSubmission.ts for purpose of demonstrating shifted encryption boundary.
```

Comment:
/nit can update the comment, this is no longer reference implementation :)

Reply:
sorry, just realised that these things from the reference implementation can be refactored to reuse what email submission already has! give me a moment

Reply:
refactored here [a0ea001](https://github.com/opengovsg/FormSG/pull/6587/commits/a0ea001e4208d4a0e639d545a36fbdc88e1e262c)!